drivers/gpio/gpio_stm32.c

```diff
@@ -579,12 +585,13 @@ static int gpio_stm32_get_config(const struct device *dev,
 	otype = LL_GPIO_GetPinOutputType(gpio, pin_ll);
 	pupd = LL_GPIO_GetPinPull(gpio, pin_ll);
 	mode = LL_GPIO_GetPinMode(gpio, pin_ll);
+	set = LL_GPIO_IsOutputPinSet(gpio, pin_ll);
```

Comment:
Pin could be "set" on IDR as well. Please use something more explicit, like 'out_set', 'oset', 'output_set', ...

Reply:
using a structure instead